createdb.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python3
+import sys
+
+from securedrop_client.models import Base, make_engine
+Base.metadata.create_all(make_engine(sys.argv[1]))
```

Comment:
I think the comment got lost, but I still think this is better as a here-doc since it's just for the dev env and it's simple enough to just shove into the bash script.